src/Geosuggest.jsx

```diff
@@ -201,20 +226,20 @@ class Geosuggest extends React.Component {
     }
 
     this.setState({isLoading: true}, () => {
-      this.autocompleteService.getPlacePredictions(
-        options,
-        suggestsGoogle => {
-          this.setState({isLoading: false});
-          this.updateSuggests(suggestsGoogle || [], // can be null
-            () => {
-              if (this.props.autoActivateFirstSuggest &&
-                !this.state.activeSuggest
-              ) {
-                this.activateSuggest('next');
-              }
-            });
-        }
-      );
+      this.autocompleteService.getPlacePredictions(options, suggestsGoogle => {
+        this.setState({isLoading: false});
+        this.updateSuggests(
+          suggestsGoogle || [], // can be null
+          () => {
```

Comment:
Move this function to a method and avoid creating a new function on every `searchSuggest` call.

Reply:
This would be a refactoring, not just a tooling upgrade.